pex/compatibility.py

```diff
@@ -108,13 +108,15 @@ def exec_function(ast, globals_map):
 
     from urllib.error import HTTPError as HTTPError
     from urllib.request import build_opener as build_opener
+    from urllib.request import FileHandler as FileHandler
```

Comment:
N.B.: On general Pex requirements for the uninitiated:
1. It still supports Python 2.7
2. It ships as both a distribution on PyPI and a Pex PEX via github releases.

2 makes it extremely unattractive to vendor requests since its platform-specific and the shipped Pex PEX - which is currently pure python - would need to grow ultiplatform support that included {linux wheels, and two flavors of macOS wheels} X {2.7, 3.5...3.9}.